src/Config/CS_SDK.props

```diff
@@ -40,8 +39,17 @@
     <None Remove="obj\**" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="$(SolutionDir)AssemblySharedInfoGenerator\AssemblySharedInfo.cs">
+    <Compile Condition="'$(ProjectName)'!='AssemblyInfoGenerator'" Include="$(SolutionDir)AssemblySharedInfoGenerator\AssemblySharedInfo.cs">
       <Link>AssemblySharedInfo.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemDefinitionGroup>
+    <ProjectReference>
+      <Private>False</Private>
```

Comment:
this applies to which project reference?

Reply:
This will be the default value for all project references
Any project that specifies differently will act as an override